### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by sreya92**

### CODE REVIEW

File: ci/ensuremod.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# inspired by nhooyr's days as CI overlord
```
Comment (nhooyr) [inline ci/ensuremod.sh]: 🔥

**Comment by nhooyr** [comment]
who said I retired as ci overlord????

**Comment by sreya92** [comment]
you're a proxy boy now

**Comment by nhooyr** [comment]
no dear sir, I'm Mr. WebSocket himself

**Comment by sreya92** [comment]
#nhooyr_software

**Pull request merged**